src/app.ts

```diff
@@ -406,7 +406,7 @@ const initializeAppModules = () => {
       },
     ],
     action: function (taskInfo, output) {
-      childWindow = microsoftTeams.dialog.url.open(taskInfo);
+      childWindow = microsoftTeams.tasks.startTask(taskInfo);

```

Comment:
Why switch back to the v1 API here?

Reply:
I noticed that this change made for a very small modal instead of the normal startTask modal. I didn't want to inadvertently alter any UI functionality.